src/cc/lib/distributed/server.cc

```diff
@@ -83,6 +83,12 @@ class EmptyGraphEngine final : public snark::GraphEngine::Service
         return grpc::Status::OK;
     }
 
+    grpc::Status GetNeighborCounts(::grpc::ServerContext *context, const snark::GetNeighborsRequest *request,
+                                   snark::GetNeighborCountsReply *response) override
+    {
+        return grpc::Status::OK;
```

Comment:
I don't think this implementation will work with the `GRPCClient` defined in `client.cc`. That client expects response counts length to be equal to the number of nodes, but you don't return back anything. Please, modify client.cc and add a test that uses both `GraphEngineServiceImpl` and `EmptyGraphEngine` classes that are connected to a single client.